### PR TITLE
Feature/audiofile list

### PR DESCRIPTION
**Pull request by K-Juno**

### CODE REVIEW

File: components/Cloudinary.js [hunk 1]
```diff
@@ -20,24 +18,53 @@ export default function CloudinaryUpload() {
             'upload_preset',
             process.env.NEXT_PUBLIC_UPLOAD_PRESET
           );
+
+          if (audioValue === '') {
+            toast('Your haven`t selected a file!', {
```
Comment (DoroVernier) [inline components/Cloudinary.js]:
```suggestion
            toast('You haven`t selected a file!', {
```

File: components/Cloudinary.js [hunk 2]
```diff
@@ -20,24 +18,53 @@ export default function CloudinaryUpload() {
...
-            // }
+            `https://api.cloudinary.com/v1_1/${process.env.NEXT_PUBLIC_CLOUDNAME}/upload`,
+            {
+              method: 'POST',
+              body: formData,
+            }
           );
           const json = await response.json();
           console.log(json);
+
```
Comment (DoroVernier) [inline components/Cloudinary.js:48]: console.log kann weg

File: components/Cloudinary.js [hunk 3]
```diff
@@ -20,24 +18,53 @@ export default function CloudinaryUpload() {
...
-            //   body: formData,
-            // }
+            `https://api.cloudinary.com/v1_1/${process.env.NEXT_PUBLIC_CLOUDNAME}/upload`,
+            {
+              method: 'POST',
+              body: formData,
+            }
           );
           const json = await response.json();
           console.log(json);
```
Comment (DoroVernier) [inline components/Cloudinary.js:47]:
```suggestion
 
```

Comment (DoroVernier) [inline components/Cloudinary.js:46]:
```suggestion

```

File: pages/recordings/index.js [hunk 4]
```diff
@@ -1,11 +1,60 @@
...
+} from '../../components/Styling';
 import CloudinaryUpload from '../../components/Cloudinary';
-import { PageTitle } from '../../components/Styling';
+import styled from 'styled-components';
+import getAllAudio from '../../services/audioService';
+import { useState } from 'react';
+
+export async function getServerSideProps() {
+  const audio = await getAllAudio();
+  console.log(audio);
```
Comment (DoroVernier) [inline pages/recordings/index.js]:
```suggestion

```

**Review by DoroVernier — COMMENTED** [review]
Some minor comments, other than that: Great Work!

File: pages/recordings/index.js (hunk 4 above)
Comment (K-Juno) [reply pages/recordings/index.js]: I'll remove it!

File: components/Cloudinary.js (hunk 3 above)
Comment (K-Juno) [reply components/Cloudinary.js:46]: Actually this one's kinda helpful/necessary in the Cloudinary upload process.

File: components/Cloudinary.js [hunk 5]
```diff
@@ -20,24 +17,47 @@ export default function CloudinaryUpload() {
             'upload_preset',
             process.env.NEXT_PUBLIC_UPLOAD_PRESET
           );
+
+          if (audioValue === '') {
+            toast('Your haven`t selected a file!', {
```
Comment (DanielGennat) [inline components/Cloudinary.js]: "You" :)

**Review by DanielGennat — APPROVED** [review]

**Pull request merged**